README.md

```diff
@@ -232,43 +232,26 @@ You can download individual policies directly:
 conftest pull instrumenta.azurecr.io/test
 ```
 
-Policies are stored in OCI-compatible registries. You can read more about this idea in
-[this post](https://stevelasker.blog/2019/01/25/cloud-native-artifact-stores-evolve-from-container-registries/).
-
-If you have a compatible OCI registry you can also push new policy bundles like so:
+Pull also supports other policy locations, such as git or https. Under the hood conftest leverages [go-getter](https://github.com/hashicorp/go-getter) to download policies. For example, to download a policy via https:
 
 ```console
-conftest push instrumenta.azurecr.io/test
+conftest pull https://raw.githubusercontent.com/instrumenta/conftest/master/examples/compose/policy/deny.rego
 ```
 
-`conftest` also supports a simple configuration file which can be used to store the
```

Comment:
are we no longer supporting `configfile` ? I see we do. Why did we remove this part from `README`?

Reply:
I removed the update command in this PR as I thought it would be outdated. But for backwards compatibility I can bring it back and change the flag for updating from an URL to something else (`update-from`? Any suggestions?). Let me know what you think.

Reply:
But we still use the configuration:
```toml
policy = "test"
namespace = "conftest"
``` 
defined in `examples/configfile` (for example: Removing it causes the acceptances to fail)
Therefore, I guess we should still document the `configfile`. What do you think?